apis/r/src/rinterface.cpp

```diff
@@ -434,6 +434,114 @@ bool c_allows_dups(const std::string& uri, Rcpp::XPtr<somactx_wrap_t> ctxxp) {
     return sch->allows_dups();
 }
 
+// Taken from tiledb-r
```

Comment:
Just a thought, maybe put this in `apis/r/src/rutilities.cpp`

Reply:
Moved in b7b004e